src/main/kotlin/net/maizegenetics/phgv2/agc/AnnotateFastas.kt

```diff
@@ -125,14 +126,10 @@ class AnnotateFastas : CliktCommand(help = "Annotate FASTA files with sample nam
                 // written to the outputDir
                 // if  the line starts with >, append " sampleName=${sampleName}" to the line
 
-                // if file was previously compressed, we compress the new file.
-                // Strip off the .gz extension before writing the new non-compressed file.
-                // it wil be added when we re-compress the file.
-                // The file names will either be <sampleName>.fa or <sampleName>.gz
-                // When the file is compressed, we only add .gz as an extension as AGC
-                // takes everything before the last period as the sample name and we
-                // want the sample name to look like "B73" or "CML247" and not "B73.fa" or "CML247.fa"
-                val justName = if (fastaFile.endsWith(".gz")) sampleName else "${sampleName}.fa"
+                // Previously if the original file was previously compressed, we compressed the new file.
```

Comment:
Should this say wasn't?

Reply:
No, it is correct other than I should remove one of the "previously".

In the original code:  if the original fasta file was compressed, I compressed the annotated fasta.  If the original was uncompressed, I did not compress the created file.  

Now I am never compressing the annotated files as these files are input for align-assemblies and anchorwave does not handle compressed files.